cloudvolume/exceptions.py

```diff
@@ -124,4 +124,12 @@ class SpecViolation(Exception):
 
 class SpatialIndexGapError(Exception):
   """Part of the spatial index was not found. A complete result set cannot be fetched."""
+  pass
+
+class WriteLockAcquisitionError(Exception):
```

Comment:
You might want to think about inheritance for your exceptions, so that it's easier to catch groups of exceptions

Reply:
This is a good idea. I'm not sure what the right hierarchy is right now though. It would be helpful to have a use case.